PusherClient/EventEmitter.cs

```diff
@@ -16,7 +16,9 @@ public void Bind(string eventName, Action<dynamic> listener)
         {
             if(_eventListeners.ContainsKey(eventName))
             {
-                _eventListeners[eventName].Add(listener);
+                // To avoid triggering the same bind multiple times

```

Comment:
In my case (as also on ExampleApplication), I have a code like this on Subscribed event.

```
static void _chatChannel_Subscribed(object sender)
{
    Console.WriteLine("Hi! Type 'quit' to exit, otherwise type anything to chat!");

    chatChannel.Bind("my-message", (dynamic data) =>
    {
        Console.WriteLine("[" + data.name + "] " + data.message);
    });
}
``` 

When some connection failure occurs, all channels are subscribed again. Which means that the Subscribed event can be triggered infinite times. Every time this happens a new bind is created for the same method. So, when a message is received it triggers the event/method multlipe times. 

In this particulare example, it writes the same message multiple times on the console 

Should I remove this comment? Explain this in details?